Assets/Tests/InputSystem/CoreTests_Editor.cs

```diff
@@ -676,6 +676,41 @@ public void Editor_InputAsset_CanReplaceBindingGroupThroughSerialization()
         Assert.That(map.bindings[0].groups, Is.EqualTo(""));
     }
 
+    [Test]
+    [Category("Editor")]
+    public void Editor_InputAsset_AddingActionToAssetWithMultipleMaps()
```

Comment:
This behavior is not specific to the editor. Should be in `CoreTests_Actions`.

Also, the test name is just a name and doesn't describe behavior. Something `Actions_AddingActionToAssetWithEnabledActions_ThrowsException`.

Reply:
Ah sorry, missed the fact that CoreTests_Actions had InputActionAsset related tests. Fixed.